accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -3356,6 +3411,10 @@ Here is one way it could be broken down into separate pieces:
 
 -   Handle negative length lists and maps (#2701).
 
+-   Disambiguate the grammar around bare identifiers (#2714). The overall
+    syntax and semantics are unchanged, but the grammar itself is now
+    unambiguous.
```

Comment:
I'd prefer to say that this particular ambiguity has been removed. ;-)  For instance, we have cases like `f(g<A, B>(c))` which are disambiguated using rules that aren't part of the grammar.

Reply:
Haha, fair. Done.